test/simulator/abilities/aftermath.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const assert = require('./../../assert');
+const common = require('./../../common');
+
+let battle;
+
+describe('Atermath', function () {
+	afterEach(function () {
+		battle.destroy();
+	});
+
+	it("should hurt attackers by 1/4 their max HP when this Pokemon is KOed by a contact move", function () {
+		battle = common.createBattle();
+		battle.setPlayer('p1', {team: [{species: 'Voltorb', level: 1, item: 'rockyhelmet', ability: 'aftermath', moves: ['sleeptalk']}]});
```

Comment:
The reason Rocky Helmet isn't in its own test is that I'm actually testing if they can work on the same hit, because it's possible there may be a bug related to this.